### PR TITLE
Initial logout commit - error

### PR DESCRIPTION
**Pull request by cedricyong00**

### CODE REVIEW

**Review by natsumi-h — CHANGES_REQUESTED** [review]
On hold until error resolved

File: src/components/Layout/Header.jsx [hunk 1]
```diff
@@ -36,6 +40,26 @@ export const Header = ({ user, setUser }) => {
   const navigate = useNavigate();
   const { successToast } = useToast();
 
+  // Logout function
+  async function logOut(token, userData) {
+    const userInfo = getUser();
+    const { sendRequest } = useFetch();
+    const logoutURL = `${import.meta.env.VITE_API_URL}/logout`;
```
Comment (natsumi-h) [inline src/components/Layout/Header.jsx]: wrong endpoint url correct: `user/logout`

File: src/components/Layout/Header.jsx [hunk 2]
```diff
@@ -29,46 +29,36 @@ import classes from "./HeaderTabs.module.css";
...
-      // send request
-      const res = await sendRequest(
-        logoutURL,
+  const handleLogout = () => {
+    try {
+      const res = sendRequest(
+        `${import.meta.env.VITE_API_URL}/user/logout`,
         "POST",
-        userInfo.email
+        { email: user.email }
```
Comment (natsumi-h) [inline src/components/Layout/Header.jsx:47]: email has to be passed by object as BE API accepts

File: src/components/Layout/Header.jsx [hunk 3]
```diff
@@ -29,46 +29,36 @@ import classes from "./HeaderTabs.module.css";
 import { Link, useLocation, useNavigate } from "react-router-dom";
 import logo from "../../assets/logo-new.png";
 import useToast from "../../hooks/useToast";
-import { getUser } from "../../service/users";
 import useFetch from "../../hooks/useFetch";
-import { removeToken } from "../../util/security";
 
 export const Header = ({ user, setUser }) => {
```
Comment (natsumi-h) [inline src/components/Layout/Header.jsx:34]: user object passed by props from `Header.jsx` so simply can destruct it and pass to API body

**Review by natsumi-h — APPROVED** [review]

**Comment by natsumi-h** [comment]
Approved with fix as commented.
Plus sorry let me push some minor bug fix together..

@jx0906 
Please approve!

**Review by jx0906 — APPROVED** [review]

**Pull request merged**